### DISCUSSION TITLE
Alternative to `RenderForm` and `RenderLoop` in Vortice?

### DISCUSSION THREAD

**Discussion by lostmsu** [amerkoleci/Vortice.Windows#432] [Q&A] [answered]
SharpDX had these two useful classes that make getting started very easy: `RenderForm` and `RenderLoop`. Are there good alternatives in/for Vortice bindings?

**Comment by amerkoleci** [comment]
Yea, could be good additions.
Are you gonna make PR?

Cheers

**Reply by amerkoleci** [reply, marked as answer]
Just added Vortice.WinForms and Vortice.Wpf for integration with WindowsForms and Wpf.

Cheers
